.github/workflows/test.yml

```diff
@@ -52,11 +52,29 @@ jobs:
       - name: run checks & tests
         run: RUST_MATRIX=${{ matrix.rust }} cargo xtask run-checks ${{ matrix.test }}
 
+      - name: Install Grcov
+        run: cargo install grcov
+
+      - name: Run grcov
+        run: grcov . --binary-path target/debug/deps/ -s . -t cobertura --branch --ignore-not-existing --ignore 'target/**' --ignore '../**' --ignore '/*' -o coverage.xml
```

Comment:
We can upload the coverage results on [codecov](https://about.codecov.io/) that better shows which lines of code are still not covered by tests, so we can replace `coverage.xml` with `lcov.info` and add `-t lcov` instead of `-t cobertura`. Perhaps we can also replace `target/debug/deps` with only `target/debug/`. We can also decide to exclude some directories from code coverage analysis adding more `--ignore path` directories. This should also speed CI

Reply:
Ok! I was using a vendorless approach, but Codecov is great. 

Is the Burn repo already set on Codecov?

Reply:
It is not necessary to set up anything, just the action is sufficient